src/test/java/com/sorts/CombSortTest.java

```diff
@@ -0,0 +1,85 @@
+package com.sorts;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CombSortTest {
+
+    //One global combSort instance variable
+    CombSort combSort = new CombSort();
+
+    //Integer Sort Test
```

Comment:
Same goes here. The naming scheme of your methods should be `testSomething`. Like `testIntegerSort` and `testStringSort`. Methods in Java start with a lowercase letter. Make sure to correct that everywhere else too.

Reply:
how about combSortIntegerTest ?

Reply:
No, just change it to `testStringSort` and `testIntegerSort`. Your class is named `CombSortTest` already, no need to add it to the method names. Alongside that, all test methods from the other classes are named using the ``testXXX` scheme. And to keep consistency it would be better if you could change them too :)

Reply:
CycleSortTest class does not follow the consistency you mentioned ?!
it follows what I suggested :)
class CycleSortTest {

    @Test
    void cycleSortIntegerTest() {

        CycleSort cycleSort = new CycleSort();

Reply:
If you take a look at all the other tests, they follow the convention `testXXX` where XXX gives some details of what is being tested. In your case, it would be `testStringSort` or something similar.

Well, looks like some tests need a rename then :) They shouldn't be named like that though. Even the JUnit site lists the `testXXX` naming scheme as a go-to.

Reply:
haha, yes now you see what I mean, and now I see what you mean, I just noticed in the /search folder, the tests follow the naming convention of testXXX.

I did the changes, did you want me to also remove all the comments like this one:
//Integer Sort Test

I'm almost done, and thank you for the review and the edit suggestions. Its always good to have another pair of eyes. I admit I totally overlooked the method names.

Reply:
Would be nice if you could remove any comments that don't really add value, yes. If they are needed to explain _why_ a certain thing is done, let them in. But something like `// Integer Sort test` is not really needed.

No problem :)

Reply:
alright, done.
I made a commit to the same pull request. 
Do you know when will pull requests be merged with the development branch ?

Reply:
It usually takes some time...